README.md

```diff
@@ -13,3 +13,5 @@ New York accident rate
 <br/>
 
 <b>Important: </b><i>To add local git hooks you have to execute sbt writeHooks</i>
+
+Also you can see Kibana dashboards <a href="http://192.168.8.205:5601/app/kibana#/dashboard/d525be90-7b0e-11e9-a32a-af1afc57cb10?_g=()&_a=(description:'',filters:!(),fullScreenMode:!f,options:(darkTheme:!f,hidePanelTitles:!f,useMargins:!t),panels:!((embeddableConfig:(),gridData:(h:15,i:'1',w:24,x:0,y:0),id:e98348f0-7b0c-11e9-a32a-af1afc57cb10,panelIndex:'1',type:visualization,version:'6.7.2'),(embeddableConfig:(),gridData:(h:14,i:'6',w:12,x:25,y:15),id:ace9f880-7afd-11e9-beb2-dd205b5f060a,panelIndex:'6',type:visualization,version:'6.7.2'),(embeddableConfig:(),gridData:(h:15,i:'8',w:11,x:37,y:15),id:'7179ff50-7b0e-11e9-a32a-af1afc57cb10',panelIndex:'8',type:visualization,version:'6.7.2'),(embeddableConfig:(),gridData:(h:14,i:'9',w:14,x:0,y:15),id:'73d3e2f0-7b10-11e9-a32a-af1afc57cb10',panelIndex:'9',type:visualization,version:'6.7.2'),(embeddableConfig:(),gridData:(h:15,i:'10',w:24,x:24,y:0),id:d2aec8b0-7b9e-11e9-bf53-9d1dd598665d,panelIndex:'10',type:visualization,version:'6.7.2'),(embeddableConfig:(),gridData:(h:15,i:'11',w:11,x:14,y:15),id:'54c2e5b0-7ba5-11e9-bf53-9d1dd598665d',panelIndex:'11',type:visualization,version:'6.7.2')),query:(language:lucene,query:''),timeRestore:!f,title:percentage_data_nycollisions,viewMode:view)">here</a>
```

Comment:
Let's make **dashboards** as link. I think **here** is redundant.

Reply:
done